spec/lib/tasks/change_history_spec.rb

```diff
@@ -0,0 +1,112 @@
+RSpec.describe "Change History tasks" do
+  include ActiveJob::TestHelper
+
+  before do
+    populate_default_government_bulk_data
+  end
+
+  describe "change_history:show" do
+    let(:change_history_entry) do
+      {
+        "id": SecureRandom.uuid,
+        "public_timestamp": "2020-02-20T16:00:00+00:00",
+        "note": "Test",
+      }
+    end
+    let(:first_edition) do
+      create(:edition,
+             current: false,
+             first_published_at: "2020-02-20 12:00:00",
+             locale: "en")
+    end
+    let(:second_edition) do
+      create(:edition,
+             document: first_edition.document,
+             number: 2,
+             locale: "en",
+             change_history: [change_history_entry])
+    end
```

Comment:
Again not too sure on need for second edition here

Reply:
Will fix.